releases/release-5.0.1.md

```diff
@@ -0,0 +1,88 @@
+---
+title: TiDB 5.0.1 Release Notes
+---
+
+# TiDB 5.0.1 Release Notes
+
+Release date: April 23, 2021
+
+TiDB version: 5.0.1
+
+## Improvements
+
++ TiKV
+
+    - Use `zstd` to compress the Region snapshot [#10005](https://github.com/tikv/tikv/pull/10005)
+
++ PD
+
+    - Modify the Region score calculator to better satisfy isomerous stores [#3605](https://github.com/pingcap/pd/pull/3605)
+    - Avoid the unexpected statistics after adding the `scatter region` scheduler [#3602](https://github.com/pingcap/pd/pull/3602)
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - Remove some might-be-misunderstood information in the summary log [#1009](https://github.com/pingcap/br/pull/1009)
+
+## Bug Fixes
+
++ TiDB
+
+    - Fix the issue that the execution result of project elimination might be wrong when the projection result is empty [#24093](https://github.com/pingcap/tidb/pull/24093)
+    - Fix the issue of wrong query results when the column contains `NULL` values in some cases [#24063](https://github.com/pingcap/tidb/pull/24063)
+    - Forbid generating MPP plans when the scan contains virtual columns [#24058](https://github.com/pingcap/tidb/pull/24058)
+    - Fix the wrong reuse of `PointGet` and `TableDual` in Plan Cache [#24043](https://github.com/pingcap/tidb/pull/24043)
+    - ping @eurekaka @winoros: Append common handle columns into the schema of index merge table plan [#24042](https://github.com/pingcap/tidb/pull/24042)
+    - Fix the type inference of the BIT-type errors [#24027](https://github.com/pingcap/tidb/pull/24027)
+    - Fix the issue that some optimizer hints do not take effect when the `PointGet` operator exists [#23685](https://github.com/pingcap/tidb/pull/23685)
+    - ping @winoros @AilinKid: Fix the cases that DDL would parse the args failed when converting job status to rolling back [#24080](https://github.com/pingcap/tidb/pull/24080)
```

Comment:
@eurekaka @AilinKid For this note, there is a remarkable difference between the English version and the Chinese version. **Neither version** reads through. **Neither version** syntactically makes sense. **Neither version** is within my capacity to comprehend and to make decisions. Please take a look, fix the note in both languages, and make sure that the two versions are literally consistent.

The Chinese version: ``` 修复某些情况下 DDL Job 在将状态设置外 rolling back 时能出现的参数解析失败问题 [#24080](https://github.com/pingcap/tidb/pull/24080) ```